meta-balena-common/conf/distro/include/balena-os.inc

```diff
@@ -36,6 +36,9 @@ DISTRO_FEATURES_DEFAULT_remove = "x11"
 DEVELOPMENT_IMAGE ?= "0"
 DISTRO_FEATURES_append = " ${@bb.utils.contains('DEVELOPMENT_IMAGE','1','development-image','',d)}"
 
+# Default u-boot delay for development images
+OS_DEV_UBOOT_DELAY ?= "-2"
```

Comment:
There is only one recipe using it so I'd keep it simple for now and only define the default in the uboot recipe. Also, I'd document this new variable.

Reply:
@agherzan Unless I'm mistaken, edits to `build/local.conf` can't tweak recipe specific variables.

Reply:
That is true. What I was proposing was to move this default from distro to the uboot recipe.